docs/api/history.md

```diff
@@ -67,6 +68,27 @@ In the example above, the following tree would be produced:
 branch is used to determine state. Here, a Microcosm's state will be
 the result of dispatching `root`, `one`, and `three` to Domains.
 
+### Reconciling
+
+What makes History such a powerful feature of Microcosm is it's
+ability to walk through recorded actions whenever there is a change,
+similar to a rebase in git.
+
+Let's say we have four actions pushed to our History, and that three
+of the four are completed `(C)`, but the second action we pushed is
+taking a long time and is still open `(O)`.
+```
+[root] - [one](C) - [two](O) - [three](C) - [four](C)
+```
+
+When our `two` action here resolves or rejects, History will initiate
+a reconciling starting at `two`, and walking forward to `three` and
+then `four`. At each step, your Domains will recompute changes to the
```

Comment:
technically they're just things that mutate the repo's state, think i'll leave it at "the state"